src/App.jsx

```diff
@@ -3,26 +3,102 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { request } from './api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    inputSearchValue: '',
+    todosStatus: 'All',
```

Comment:
no need to save these properties in the App component state, move them to TodoList component